### PR TITLE
clean up shell script lint

### PR DESCRIPTION
**Pull request by reidpr**
Addresses #577 and #580.

This PR adds some shell scripts to lint that were missed previously, so there are a few adjustments throughout.

### CODE REVIEW

File: test/Build.centos7xz [hunk 1]
```diff
@@ -19,9 +19,9 @@
 
 set -ex
 
-srcdir=$1
+#srcdir=$1  # unused
```
Comment (rstyd) [inline test/Build.centos7xz]: What's the reasoning for leaving these variables commented out instead of removing them?

File: test/build.bats [hunk 2]
```diff
@@ -67,60 +67,82 @@ load common
...
+                   \(    -name .git \
+                      -o -name build-aux \) -prune \
+                -o \( -name '*.sh' -print \) \
+                -o \( -name '*.bash' -print \) \
+                -o \( -type f -exec awk '/^#!\/bin\/(ba)?sh/ {print FILENAME}
+                                         {nextfile}' {} + \) )
+    # Bats scripts. Use sed to do two things:
+    #
+    # 1. Make parseable by ShellCheck by removing "@test '...'". This does
+    #    remove the test names, but line numbers are still valid.
```
Comment (rstyd) [inline test/build.bats]: What is the effect of removing the test names?

File: test/build.bats [hunk 3]
```diff
@@ -67,60 +67,82 @@ load common
...
-    done
-    # BATS scripts
+    done < <( find "$ch_base" \
+                   \(    -name .git \
+                      -o -name build-aux \) -prune \
+                -o \( -name '*.sh' -print \) \
+                -o \( -name '*.bash' -print \) \
+                -o \( -type f -exec awk '/^#!\/bin\/(ba)?sh/ {print FILENAME}
+                                         {nextfile}' {} + \) )
+    # Bats scripts. Use sed to do two things:
```
Comment (j-ogas) [inline test/build.bats]: Can we align the indention to line up 'name' (the common argument) as we do in `ch-test`?

File: test/Build.centos7xz (hunk 1 above)
Comment (reidpr) [reply test/Build.centos7xz]: To be consistent, i.e., always mention all three arguments in the same order. Commenting them out satisfies ShellCheck.

File: test/build.bats (hunk 2 above)
Comment (reidpr) [reply test/build.bats]: ShellCheck warnings will not be quite as clear. But the line numbers are correct, so you can still go look easily.

File: test/build.bats (hunk 3 above)
Comment (reidpr) [reply test/build.bats]: Well, the two groups of `-name` each line up within the group, unless I messed up. I don't quite follow. Can you write what indentation you propose?

**Pull request merged**